README.md

```diff
@@ -61,16 +61,17 @@ I'm open to more styles if they can be accomplished with simple aliases.  Open a
 ### Example
 
 ```js
-import "./testie/testie" for Testie, Assert
+import "wren-testie/testie" for Testie
```

Comment:
Do you think I should rename the base file just so the magic `import "wren-testie"` syntax works?

Reply:
I mean long-term perhaps it's weird if every package has `wren` in the name though, but if that's what the repo is named then that's the default checkout path.